src/main/java/io/github/thebusybiscuit/slimefun4/utils/ExperienceUtils.java

```diff
@@ -0,0 +1,71 @@
+package io.github.thebusybiscuit.slimefun4.utils;
+
+import javax.annotation.Nonnull;
+import java.util.Map;
+
+/**
+ * Some helper methods for dealing with experience converting.
+ *
+ * @author kingdom84521
+ *
+ * @see io.github.thebusybiscuit.slimefun4.core.guide.SlimefunGuideImplementation
+ * @see <a href="https://minecraft.fandom.com/wiki/Experience"></a>
+ *
+ */
+public final class ExperienceUtils {
```

Comment:
Please also add javadocs to all the public methods in this class, it's a utility class after all.

Reply:
fix